embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/EmbraceImpl.java

```diff
@@ -311,7 +311,7 @@ final class EmbraceImpl {
         this.essentialServiceModuleSupplier = essentialServiceModuleSupplier;
         this.dataCaptureServiceModuleSupplier = dataCaptureServiceModuleSupplier;
         this.deliveryModuleSupplier = deliveryModuleSupplier;
-        this.tracer = LazyKt.lazy(() -> new EmbraceTracer(initModule.getSpansService()));
+        this.tracer = initModule.getTracer();
```

Comment:
This is needed almost immediately now that spans is always on, so there's no reason to make this lazy.